packages/zowe-explorer-api/src/profiles/ZoweExplorerApi.ts

```diff
@@ -648,5 +648,17 @@ export namespace ZoweExplorerApi {
          * @returns {string[]}
          */
         registeredApiTypes(): string[];
+
+        /**
+         * Register a callback function to be used when a change to the team config profile is performed.
+         * @param callback
+         */
+        registerProfileChangeCallback(callback: (eventType: EventTypes) => Promise<void>): void;
+
+        /**
+         * Get function that is called when a change to the team config profile is performed.
+         * @returns Function
+         */
+        getProfileChangeCallback(): (eventType: EventTypes) => Promise<void>;
```

Comment:
@JillieBeanSim do you know if this would be an issue? I think if we make it optional is fine by me too.

Reply:
Since `IApiRegisterClient` is exported, this would technically be a breaking change for extenders that implement the `IApiRegisterClient` interface, but I think updating the functions to be optional would be sufficient 😋